src/stdatamodels/model_base.py

```diff
@@ -117,11 +137,14 @@ def __init__(self, init=None, schema=None, memmap=False,
 
         # Load the schema files
         if schema is None:
-            # Create an AsdfFile so we can use its resolver for loading schemas
-            asdf_file = AsdfFile()
-            schema = asdf_schema.load_schema(self.schema_url,
-                                             resolver=asdf_file.resolver,
-                                             resolve_references=True)
+            if self.schema_url is None:
+                schema = _DEFAULT_SCHEMA
+            else:
+                # Create an AsdfFile so we can use its resolver for loading schemas
+                asdf_file = AsdfFile()
+                schema = asdf_schema.load_schema(self.schema_url,
+                                                resolver=asdf_file.resolver,
+                                                resolve_references=True)
```

Comment:
This can probably be switched to the new way
```suggestion
                schema = asdf_schema.load_schema(self.schema_url, resolve_references=True)
```
as now the default resolver pulls in all the installed extensions, right?

Reply:
Yes, that's the better way for sure